ProjectLighthouse/Administration/Maintenance/Commands/QueueGlobalNotification.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using LBPUnion.ProjectLighthouse.Database;
+using LBPUnion.ProjectLighthouse.Logging;
+using LBPUnion.ProjectLighthouse.Types.Logging;
+using LBPUnion.ProjectLighthouse.Types.Maintenance;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace LBPUnion.ProjectLighthouse.Administration.Maintenance.Commands;
+
+public class QueueGlobalNotification : ICommand
```

Comment:
Is this intended to be able to deliver notifications to everybody? Otherwise this class should probably renamed